packages/perseus/src/widgets/__tests__/radio/base-radio_test.jsx

```diff
@@ -201,14 +208,54 @@ describe("base-radio", () => {
 
             // Act
             userEvent.click(
-                screen.getByRole("checkbox", {name: "Select Choice C"}),
+                screen.getByRole("radio", {name: "(Choice C) Option Gamma"}),
             );
 
             // Assert
             expect(updatedValues).toMatchObject({
                 checked: [false, false, true, false],
             });
         });
+
+        it("deselects previously selected choice", () => {
```

Comment:
Yes, this is because you cannot deselect a native radio element by clicking it directly. I wanted the test to confirm that when you select a different option after having selected one already, that the previously selected choice is no longer selected.

Reply:
I see, I think "changes selected choice" would be more accurate then since "deselected" is now kind of a loaded term in the Radio Widget.

Reply:
Updated :)